### PR TITLE
sw-6690 fixing flaky tests #minor

### PR DESCRIPTION
**Pull request by Nicholassully**

### CODE REVIEW

File: cypress/e2e/supervision/clients/client-tasks-tab.cy.js [hunk 1]
```diff
@@ -35,7 +35,7 @@ describe(
       cy.get('.details-row > td').should('contain.text', 'A client has been created');
 
       cy.get('.order-number').should('contain.text', 'PFA');
-
+      cy.wait(500);
```
Comment (denialanderror) [inline cypress/e2e/supervision/clients/client-tasks-tab.cy.js:38]: What is this waiting for? Is it not possible to ensure the page is in the correct state by asserting on an element?

File: cypress/e2e/supervision/orders/create-order.cy.js [hunk 2]
```diff
@@ -13,40 +13,40 @@ const createOrder = (orderType, orderSubType, orderDate, optional) => {
       .closest(".fieldset")
       .find("Select")
       .select(orderSubType);
-    cy.get('input[name="courtReference"]').should("not.be.disabled").clear();
+    cy.get('input[name="courtReference"]').should("not.be.disabled").clear({ force: true });
     cy.get('input[name="courtReference"]').type(caseRecNumber);
```
Comment (kate-49) [inline cypress/e2e/supervision/orders/create-order.cy.js:17]: Do every single one of these need a force true? I thought it was usually for when we want to target disabled elements etc. According to Cypress forcing is for when `it's not worth trying to "act like a user" to get a robot to do the exact steps a user would to interact with an element.` But I do feel like with this step we're trying to mimic a user's steps?
Would putting a wait x amount of time for an element to be visible work here?

Comment (denialanderror) [reply cypress/e2e/supervision/orders/create-order.cy.js:17]: Agreed. We shouldn't really be using workarounds unless they are absolutely necessary, and if they are, we should understand what the issue we are trying to solve by doing so.

File: cypress/e2e/supervision/clients/client-tasks-tab.cy.js (hunk 1 above)
Comment (Nicholassully) [reply cypress/e2e/supervision/clients/client-tasks-tab.cy.js:38]: To make sure that angular has finished loading everything on the pages

Comment (denialanderror) [reply cypress/e2e/supervision/clients/client-tasks-tab.cy.js:38]: Is there an element or class it can wait on instead then?

Comment (Nicholassully) [reply cypress/e2e/supervision/clients/client-tasks-tab.cy.js:38]: Its the case that it looks like it clicks the button but then other api calls come back and override the button click so never gets to that page it needs.

**Review by gregtyler — APPROVED** [review]
Approved to unblock the pipeline, but I don't know the Supervision UI well enough to provide more critical feedback

**Pull request merged**